packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -31,6 +31,51 @@ import LocusDeltaParser from './parser';
 import Metrics from '../metrics';
 import BEHAVIORAL_METRICS from '../metrics/constants';
 
+export type LocusDTO = {
```

Comment:
yes, believe it or not up until now we haven't had a type defined for Locus DTO in the meetings SDK. This is just a start, not fully populated and with many `any`s but at least it's something that we can build upon